win32/src/win32rasmodule.cpp

```diff
@@ -715,7 +715,7 @@ static PyObject *PyRasGetErrorString(PyObject *self, PyObject *args)
 
     TCHAR buf[512];
     // @pyseeapi RasGetErrorString
-    if (rc = RasGetErrorString(error, buf, sizeof(buf) / sizeof(buf[0])))
+    if (rc == RasGetErrorString(error, buf, sizeof(buf) / sizeof(buf[0])))
```

Comment:
All of these changes look wrong - the intent is that rc be passed to `ReturnRasError` - IOW, this block should read:

```
    if (rc = RasGetErrorString(error, buf, sizeof(buf) / sizeof(buf[0])))
        return ReturnRasError("RasGetErrorString", rc);
```

(it's not clear that's actually necessary as `ReturnRasError()` will call `GetLastError()` when we don't pass the error code, but it's probably safer to not rely on `GetLastError()` when we don't need to)

Reply:
gcc errors out because rc is not initialized.. must add 'int rc = ...' then..

Reply:
That seems back to front - that line *is* initializing. I'd expect the error with `==` where there's no initialization but instead you would be evaluating an uninitialized variable.

Reply:
ok, my bad.. I had reverted it back..